README.md

```diff
@@ -31,6 +31,9 @@ This provider's versions are able to install and manage the following versions o
 |-|-|-|
 |OpenStack Provider v1alpha1 (ea309e7f)|✓|✓|
 
+Kubeadm and Kubelet versions are defined in `spec.versions.controlPlane` and `spec.versions.kubelet` of `cmd/clusterctl/examples/openstack/machines.yaml.template` respectively.
```

Comment:
Maybe we should rephrase to "Kubernetes versions are defined in ...". I would not mention kubeadm here because it is an implementation detail. And in our particular case, the `versions.kubelet` is also used as the version for `kubeadm`.

Reply:
seems we do have a control pane version and kubelet version ,maybe mention both of them ?

Reply:
How about this? "Kubernetes control plane and Kubelet versions are defined in `spec.versions.controlPlane` and `spec.versions.kubelet` of `cmd/clusterctl/examples/openstack/machines.yaml.template` respectively."